tests/Integration/Infrastructure/Controller/Regulation/Fragments/GetMeasureControllerTest.php

```diff
@@ -48,7 +48,7 @@ public function testGetWithComplexVehicles(): void
 
         $this->assertSame('Circulation interdite', $measure1Header->filter('h3')->text());
         $this->assertSame('pour les véhicules de plus de 3,5 tonnes, 12 mètres de long ou 2,4 mètres de haut, matières dangereuses, Crit\'Air 4 et Crit\'Air 5, sauf piétons, véhicules d\'urgence et convois exceptionnels', $measure1Content->filter('li')->eq(0)->text());
-        $this->assertSame('du 10/03/2023 à 00h00 au 20/03/2023 à 23h59', $measure1Content->filter('li')->eq(1)->text());
+        $this->assertSame('du 10/03/2023 à 00h00 au 20/03/2023 à 23h59 et du 28/03/2023 à 08h00 au 28/03/2023 à 22h00', $measure1Content->filter('li')->eq(1)->text());
```

Comment:
Sans le correctif dans period.html.twig ça aurait affiché "... à 23h59du 28/03/2023 ..."

```suggestion
        $this->assertSame('du 10/03/2023 à 00h00 au 20/03/2023 à 23h59 et du 28/03/2023 à 08h00 au 28/03/2023 à 22h00', $measure1Content->filter('li')->eq(1)->text());
```